quickstarts/conviva/config.yml

```
@@ -0,0 +1,78 @@
+# Sets the URL name of the quickstart on public I/O (required)
+slug: conviva
+
+# Displayed in the UI (required)
+title: NRI Conviva Integration
```

See the comment(s) in the other file.

```suggestion
title: Conviva
```

---

I am perfectly fine with just `Conviva`.  For future reference, NRI == New Relic Infrastructure.